src/main/kotlin/racing_car/domain/car/Engine.kt

```diff
@@ -0,0 +1,5 @@
+package racing_car.domain.car
+
+interface Engine {
+    fun getMoveDistance(): Int
+}
```

Comment:
추상 함수가 하나인 인터페이스([SAM](https://kotlinlang.org/docs/fun-interfaces.html#sam-conversions)) 는 함수형 인터페이스로 선언한다면 람다식으로 편리하게 표현할 수 있어요.
```suggestion
fun interface Engine {
    fun getMoveDistance(): Int
}

// 사용
val fixedMoveEngine: Engine = Engine { 1 }
```

Reply:
오오 신기하네요!